koa-crud/src/data-access/mongoose/members/actions.ts

```diff
@@ -1,31 +1,58 @@
-import { MembersStore } from '../../../types/index';
-import { Member as MemberModel } from '../../../lib/mongoose/models/member';
+import MemberModelType, {
+  Member,
+  MemberDocument,
+} from '../../../lib/mongoose/models/member';
 
-const actions = ({ Member }: { Member: typeof MemberModel }): MembersStore => {
+type MemberInformation = { password?: Member['password'] | null } & Partial<
+  Pick<Member, 'username' | 'realName' | 'email' | 'bankAccount' | 'balance'>
+>;
+
+type MemberFilters = {
+  _id?: Member['_id'] | Record<string, any>;
+  username?: Member['username'] | Record<string, any>;
+};
```

Comment:
Why do you need this `| Record<string, any>` in both `_id` and `username`

Reply:
I need that for mongoose query operators like $gte, etc.